### ISSUE TITLE
Large model starts to overfit after 40k steps at a BLEU score of <3? / Will this converge with more days of training?

### ISSUE THREAD

**Issue by matthias-samwald** [google/seq2seq#197]
I'd like to validate the 'Large Model' and variations thereof with the tutorial WMT data before experimenting with my own data. On a single GPU, this takes quite some time, and I am a bit puzzled with the loss curve I get on the first day (to me it seems like it starts to overfit, screenshots attached). I'd like to know if this could be expected to turn around again with more days of training, or if something is wrong on my end.

By the way, LayerNormBasicLSTMCell seems to converge in half as many steps as the vanilla LSTMCell.

![first day of large model on wmt -- is it overfitting](https://cloud.githubusercontent.com/assets/6213601/25523824/70484dd2-2c08-11e7-9553-e9a907699414.gif)

**Comment by matthias-samwald** [comment]
In general, it would be great if other users could share the loss functions they get with their own experiments on common datasets!

**Comment by 0xymor0n** [comment]
I have the same problem

**Comment by smolendawid** [comment]
My curves look very similar to yours, but I am using my own data. I am really confused

**Comment by matthias-samwald** [comment]
@smolendawid See comments at https://github.com/google/seq2seq/issues/216. This problem seems to be caused by a recent patch.

**Comment by smolendawid** [comment]
@matthias-samwald Thank you, I'll check the previous version. But eventually you had some crashes, didn't you?

**Comment by matthias-samwald** [comment]
@smolendawid Indeed, I have not succeeded in training a model to a good BLEU score.
I have , at least for the time being, switched to https://github.com/facebookresearch/fairseq for my experiments (it seems to work very well).
